README.md

```diff
@@ -118,9 +120,9 @@ ighub, 39 New Market Rd, City Centre 434212, Onitsha
 
 
   
-# Bauchi
+## Bauchi
 * [Uplift Hub](https://uplift.ng)
-  Uplift Hub, Adajcent Admin Block, Federal Polytechnic Bauchi, Gwallameji, Bauchi 743001, Bauchi State.
+  Uplift Hub, 1st Floor Isa Yuguda House Adjacent Chartwel Hotel, Jos - Bauchi Rd, Bauchi, Nigeria
```

Comment:
Why is Jos here?

Reply:
Yes it's called Jos - Bauchi road. I attended Abubakar Tafawa Balewa University bauchi so I know the route very well sir.

Reply:
Oh okay. Thanks for the explanation. Will merge now.

Thank you!